src/redux/books/booksSlice.js

```diff
@@ -1,23 +1,43 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-const initialState = {
-  books: [],
-};
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 
 const booksSlice = createSlice({
   name: 'books',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.books.push({
-        id: action.payload.id,
-        title: action.payload.title,
-        author: action.payload.author,
-      });
+      const { title, author, category } = action.payload;
+      const newBook = {
+        item_id: `Item${state.length + 1}`,
+        title,
+        author,
+        category,
+      };
+      return [...state, newBook];
```

Comment:
- [ ] Keep doing the great work with the add and remove actions 👏 👏 👏

Reply:
@iambenkis 

- [x] I have now added an MIT.md License to this branch of the project.
- [x] Gee 😄 Thanks for the appreciation I made the changes on the `add and remove actions` while paying another attention.

I wish you have a great day.